plugins/Unity/Launcher/appdrawermodel.cpp

```diff
@@ -82,11 +84,11 @@ void AppDrawerModel::appAdded(const QString &appId)
     endInsertRows();
 }
 
-void AppDrawerModel::appRemoved(const QString &appId)
+void AppDrawerModel::appRemoved(const QString &appId, const QString &ubuntuAppId)
```

Comment:
Can you please add a comment to explain the difference between the `appId` and `ubuntuAppId`?

Reply:
Or maybe rename the second variable to `unversionedAppId`?